paper-input-container.html

```diff
@@ -515,8 +515,7 @@
     _handleValue: function(inputElement) {
       var value = this._inputElementValue;
 
-      // type="number" hack needed because this.value is empty until it's valid
```

Comment:
I might be crazy but this seems to be already broken.

Repro locally on `paper-input` `type="number"` using input `111ee` results in an empty input.

Can't repro on jsbin/don't know how to make polygit pull in the latest version.